### PR TITLE
MTP-2459 Change link icons in Menu, update Story

### PR DESCRIPTION
**Pull request by petterkoding**
Disclosures used in Header changes:

- icon should be on right side
- if disclosure is used as a link use arrow icon
- default text link styling for child links and bottom row links (underline with 3px on hover)
- add hover background-color and for open states
- icon background should change to white on hover/open state

image

### CODE REVIEW

**Review by lha047 — CHANGES_REQUESTED** [review]
Screenshot 2024-08-14 at 08 41 21
Talk to Emilie about how they want it

File: mt-kit/core/css/src/utilities/_layout.scss [hunk 1]
```diff
@@ -265,6 +265,42 @@ Only use if it does NOT affect accessibility
   border-bottom: $border-width-regular solid var(--mt-header-footer-border);
   z-index: var(--z-index-fixed);
   position: relative;
+
+  .dropdown-content {
```
Comment (lha047) [inline mt-kit/core/css/src/utilities/_layout.scss:269]: Dont think this is the right place to do this

**Comment by lha047** [comment]
Screenshot 2024-08-15 at 14 09 24
The hover effect looks wrong. Should find another solution

**Comment by petterkoding** [comment]
> Screenshot 2024-08-15 at 14 09 24 The hover effect looks wrong. Should find another solution

Do you mean because of the icon has white background on hover/open state?

**Review by lha047 — APPROVED** [review]
LGTM 👍

**Pull request merged**